src/jit/gtlist.h

```diff
@@ -190,6 +190,7 @@ GTNODE(SIMD       , "simd"       ,0,GTK_BINOP|GTK_EXOP)   // SIMD functions/oper
 //-----------------------------------------------------------------------------
 
 GTNODE(JTRUE      , "jmpTrue"    ,0,GTK_UNOP|GTK_NOVALUE)
+GTNODE(JCC        , "jcc"        ,0,GTK_LEAF|GTK_NOVALUE)
```

Comment:
Somewhere, presumably here, there should be a sizable comment that describes what this is and why it exists - i.e. why isn't this the same as GT_JTRUE? I would suggest adding a comment above, "The following are both conditional brnaches. GT_JTRUE has a single operand that computes a condition. GT_JCC implicitly reads the condition bits from a previous operation. The latter is allowed only in the LIR form used in the RyuJIT backend."